modules/ossm-cr-threescale-1x.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/customizing-installation-ossm.adoc
+// * service_mesh/v2x/customizing-installation-ossm.adoc
```

Comment:
It looks like there are two versions of this file, so the "include" note needs to be updated that this one only applies to OSSM v1x.

Reply:
Ok, I just removed the second of these comments (on line 4) in my recent commit.